File: 3.实用功能封装/3.1codeScanning/codeScanner/ViewController.m
//
//  ViewController.m
//  codeScanner
//
//  Created by WZH on 16/3/9.
//  Copyright © 2016年 王振海. All rights reserved.
//

#import "ViewController.h"
#import "ZHScanView.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    ZHScanView *scanf = [ZHScanView scanView];
//    ZHScanView *scanf = [ZHScanView scanViewWithFrame:CGRectMake(0, 0, self.view.bounds.size.width, self.view.bounds.size.height - 100)];
    scanf.promptMessage = @"您可以直接输入或者选择扫描二维码";
    [self.view addSubview:scanf];
    
    [scanf startScaning];
    
    [scanf outPutResult:^(NSString *result) {
        
        NSLog(@"%@",result);
        
    }];

    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Demo/Swift/29.Swfit3.0 - Swift setter设置数据/29.Swfit3.0 - Swift setter设置数据/ViewController.swift
//
//  ViewController.swift
//  29.Swfit3.0 - Swift setter设置数据
//
//  Created by WZH on 16/10/20.
//  Copyright © 2016年 limpid. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let p = Person()
        p.name = "Hayder"
        
        let lable = Demolabel(frame: CGRect(x: 20, y: 40, width: 100, height: 40))
        
        view.addSubview(lable)
        
        //将模型设置给label
        lable.person = p
    }

}

